cmake/modules/hunter_pick_scheme.cmake

```diff
@@ -39,6 +39,7 @@ function(hunter_pick_scheme)
   set(HUNTER_PACKAGE_SCHEME_DOWNLOAD "")
   set(HUNTER_PACKAGE_SCHEME_UNPACK "")
   set(HUNTER_PACKAGE_SCHEME_UNPACK_INSTALL "")
+  set(HUNTER_PACKAGE_SCHEME_UNPACK_BIN_INSTALL "")
```

Comment:
There is no need to introduce new variable, just use `HUNTER_PACKAGE_SCHEME_UNPACK_INSTALL`